Cargo.lock

```diff
@@ -31,15 +25,15 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.98"
+version = "1.0.100"
```

Comment:
More deps are upgrading than I would have expected. Are they required or was that accidental when using the env dep?

Reply:
Uhhh I just modified `soroban-env` then ran `cargo update`, so probably side-effects but everything still works :shrug: